index.html

```diff
@@ -13,30 +13,211 @@
 
 <body>
   <section class="headersec">
-    <header>
-      <nav id="menu-top">
-        <ul class="topnav">
-          <li class="topnav-logo"><a href="#">My Logo</a></li>
-          <li class="topnav-right"><a href="#">Home</a></li>
-          <li class="topnav-right"><a href="#">About</a></li>
-          <li class="topnav-right"><a href="#">Portfolio</a></li>
-          <li class="dropdownIcon"><a href="#">&#9776;</a></li>
-        </ul>
-      </nav>
-    </header>
+    <div class="topnav-logo">
+      <h4>My Logo</h4>
+    </div>
+    <div class="menu-icon-wrapper">
+      <div class="menu-icon" onclick="toggleCssMenu(this)">
+        <div class="three-line">
+          <span></span>
+          <span></span>
+          <span></span>
```

Comment:
- [ ] **OPTIONAL**
You left empty `span` tags in your code for no apparent reason. Please remove these empty tags to ensure a cleaner code